api_fhir_r4/mapping/organizationMapping.py

```diff
@@ -0,0 +1,99 @@
+from urllib.parse import urljoin
+
+from api_fhir_r4.configurations import GeneralConfiguration, R4OrganisationConfig
+from api_fhir_r4.models.imisModelEnums import ContactPointSystem
+
+
+class HealthFacilityOrganizationTypeMapping:
+    LEGAL_FORM_CODE = 'D'
+    LEGAL_FORM_DISPLAY = 'District organization'
+    LEGAL_FORM_SYSTEM = 'CodeSystem/organization-legal-form'
+    LEGAL_FORM_URL = f'{GeneralConfiguration.get_system_base_url()}StructureDefinition/organization-legal-form'
+
+    ORGANIZATION_TYPE = 'prov'
+
+    EMAIL_CONTACT_POINT_SYSTEM = ContactPointSystem.EMAIL
+    PHONE_CONTACT_POINT_SYSTEM = ContactPointSystem.PHONE
+    FAX_CONTACT_POINT_SYSTEM = ContactPointSystem.FAX
+
+    ADDRESS_LOCATION_REFERENCE_URL = \
+        f'{GeneralConfiguration.get_system_base_url()}/StructureDefinition/address-location-reference'
+
+    CONTRACT_PURPOSE = {
+        'code': 'PAYOR',
+        'system': 'http://terminology.hl7.org/CodeSystem/contactentity-type'
+    }
+
+    LEVEL_DISPLAY_MAPPING = {
+        'D': 'Dispensary',
+        'C': 'Health Centre',
+        'H': 'Hospital'
+    }
+
+    LEVEL_SYSTEM = f'{GeneralConfiguration.get_system_base_url()}/CodeSystem/organization-hf-level'
+
+
+class PolicyHolderOrganisationLegalFormMapping(object):
+    SYSTEM = urljoin(GeneralConfiguration.get_system_base_url(),
+                     R4OrganisationConfig.get_fhir_ph_organisation_legal_form_code_system())
+
+    fhir_ph_organisation_legal_form = {
```

Comment:
This should be a CodeSystem and values should be taken from PH module. Please check with Severin for the values.